packages/ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
please rename it to media-play instead of single play
better to use group prefix for organize icons and then easy to find they in all list.

and i'm not sure to use triangle with circle both in one icon. maybe use single `play` only as triangle?

Reply:
There is one icon and no any sense to split them

Reply:
Renamed

Reply:
like this svg
```
<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M7 28C6.73478 28 6.48043 27.8946 6.29289 27.7071C6.10536 27.5196 6 27.2652 6 27V4.99998C6.00008 4.82438 6.04639 4.6519 6.13429 4.49989C6.22218 4.34787 6.34855 4.22168 6.50069 4.134C6.65284 4.04632 6.82538 4.00026 7.00098 4.00043C7.17657 4.0006 7.34903 4.04701 7.501 4.13498L26.501 15.135C26.6525 15.2229 26.7783 15.3491 26.8658 15.5009C26.9532 15.6527 26.9992 15.8248 26.9992 16C26.9992 16.1752 26.9532 16.3473 26.8658 16.4991C26.7783 16.6509 26.6525 16.7771 26.501 16.865L7.501 27.865C7.34882 27.9534 7.17598 27.9999 7 28Z" fill="currentColor"/>
</svg>
```

Reply:
At this moment, I need only play icon. Why I should add another icons for potential use-cases?

Reply:
Renamed to `media-play-alt`